test/rendering/table-light-theme.render.ts

```diff
@@ -0,0 +1,67 @@
+import fs from 'fs';
```

Comment:
how similar is this file to the dark one? We should generalize stuff if it makes sense, so we don't have to change in two places

Reply:
quite similar. Yes, We should do that.

Reply:
Done